tests/subsys/fs/nvs/testcase.yaml

```diff
@@ -4,3 +4,4 @@ tests:
   filesystem.nvs_0x00:
     extra_args: DTC_OVERLAY_FILE=boards/qemu_x86_ev_0x00.overlay
     platform_whitelist: qemu_x86
+    skip: true
```

Comment:
I think @carlescufi already dealt with this one (in commit 22be0d8fd5902da11c9bf55a44012abc7c2cf1c1)